apps/client/src/common/hooks/useClientPath.ts

```diff
@@ -13,7 +13,7 @@ export const useClientPath = () => {
   // notify of client path changes
   useEffect(() => {
     //remove leading '/' from path
-    const fullPath = (pathname.startsWith('/') ? pathname.slice(1) : pathname) + search;
+    const fullPath = pathname + search;
```

Comment:
todo: do we need to remove this comment?
also, should we use a the URL object to create a path that we know is valid?

Reply:
I think we are already getting these from a URL object 
and  it only provides pathname and search separately